src/FlatButton/FlatButton.js

```diff
@@ -258,6 +263,7 @@ class FlatButton extends Component {
     return (
       <EnhancedButton
         {...other}
+        className={className}
```

Comment:
@oliviertassinari I tried removing this line from here and className does not work. Looks like it is not inside the `other` object.